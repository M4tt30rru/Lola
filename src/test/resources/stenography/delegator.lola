##Import "std.lola"
##Find
 ##Type(_type) ##Identifier(_name)(##ArgumentDeclarationList(_l)) -> ##Any(_delegator);
 ##replace ##(_type) ##(_name) (##(_l)) {##("" if _type.name.strip() == "void" else "return") ##(_delegator)(##(",".join(a._name.name for a in _l._args)));}

public Coffee makeCoffee(Milk milk, List<Bean> beans) -> wife.makeIt;
public void takeTrash(Bag bag) -> husband.throwIt;
