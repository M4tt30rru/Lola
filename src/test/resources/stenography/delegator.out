public Coffee makeCoffee(Milk milk, List<Bean> beans) {
 return wife.makeIt(milk, beans);
}
public void takeTrash(Bag bag) {
 husband.throwIt(bag);
}
